roadmap.md

```diff
@@ -0,0 +1,35 @@
+Roadmap Zombienet v2
+
+## Infra
+- Chaos testing, add examples and explore possibilities in `native` and `podman` provider
+- Add `docker` provider
+- Add `nomad` provider
+- Create helm chart to allow other use zombienet in k8s
+- Auth system to not use k8s users
+- Create GitHub Action and publish in marketplace
```

Comment:
Are you referring to npmjs here?
If yes then you can commit the following suggestion:

```suggestion
- Create GitHub Action and publish in marketplace (wip)
```